tests/test_netbox.py

```diff
@@ -34,6 +34,7 @@
 netbox:
     main:
         api_url: 'http://localhost/api/dcim/devices/'
+        api_token: '1234567890987654321234567890987654321'
```

Comment:
Let's have a commented key inside `netbox.yaml` to make it part of conf file.
e.g.
```
netbox:
     main:
         api_url: 'http://localhost/api/dcim/devices/'
        #api_token: ''
```

Reply:
Sounds good.